lib/SILGen/SILGenPoly.cpp

```diff
@@ -523,6 +523,14 @@ ManagedValue Transform::transform(ManagedValue v,
                                    SGF.getLoweredLoadableType(outputSubstType));
   }
 
+  // - block to AnyObject conversion
```

Comment:
Can you make sure this doesn't happen on Linux, where blocks do not use Swift-compatible reference counting?

Reply:
Okay, #13153.